Scala/Scala.sublime-syntax

```diff
@@ -867,6 +866,9 @@ contexts:
       scope: support.type.scala
       set: single-type-expression-tail
     - include: base-type-expression
+    - match: '(?=@)'
```

Comment:
Why not just do this in `single-type-expression-tail`?  This is how we handle the following case:

```scala
e: Foo with Bar
```